aiinfra-cluster/installation_scripts/install_cloud_ops_agent.sh

```diff
@@ -43,7 +136,42 @@ handle_debian() {
     }
 
     install_opsagent() {
-        curl -s https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh | bash -s -- --also-install
+        curl -s https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh |
+            REPO_SUFFIX=20230214-1.1.1 bash -s -- --also-install
+    }
+
+    install_dcgm() {
```

Comment:
@suffiank do you know of any failure modes of dcgm installation that we should maybe add retries for ?